### PR TITLE
Footer Added

### PR DESCRIPTION
**Pull request by dhairyabaweja**
# Description

Added footer with links of facebook github and mail.

Fixes # 6

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by dhairyabaweja** [comment]
gh link - https://dhairyabaweja.github.io/GirlsDoCode/

**Comment by dhairyabaweja** [comment]
@Anupam-dagar Ihave made the changes

File: index.html [hunk 1]
```diff
@@ -135,6 +136,22 @@ <h2>Johnson Family</h2>
...
+  <div id="footer">
+    <div class="container">
+        
+            <div class="row">
+                <div class="col-md-4"></div>
+                <div class="col-md-4"><p id="imp"><center><p class="footertext">Made with <img src="images/heart.png"> by Girls in Open Source.</p></center></p></div>
+                <div class="col-md-4 text-right" id="right">
+                  <a href="https://github.com/fossiiita"><img src="images/github.png" class="img-circle" alt="the-brains"></a>
+                  <a href="https://www.facebook.com/opencodeiiita/"><img src="images/facebook.png" class="img-circle" alt="..."></a>
+                  <a href="mailto:iit2015113@iiita.ac.in"><img src="images/mail.png" class="img-circle" alt="..."></a>
```
Comment (Anupam-dagar) [inline index.html]: change the mail in mail to here to geekhaven one. change the github link to geekhaven org one.

Comment (dhairyabaweja) [reply index.html]: @Anupam-dagar done

**Comment by Anupam-dagar** [comment]
@dhairyabaweja this PR is quite destructive, you have directly edited bootstrap classes powering the complete website. This will create a lot of issues in development of the website. Also please don't directly add a margin to the body tag. Find alternative approach. For now I am reverting this PR. Claim the points for this when you open a new PR for this again.

**Pull request merged**